src/chart.zoom.js

```diff
@@ -102,23 +108,36 @@ function zoomTimeScale(scale, zoom, center) {
 	var minDelta = newDiff * min_percent;
 	var maxDelta = newDiff * max_percent;
 
-	options.time.min = scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) + minDelta);
-	options.time.max = scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - maxDelta);
+	var newMin = scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) + minDelta);
+	options.time.min = newMin < min_limit ? min_limit : newMin;
```

Comment:
Or use moment to parse the value, convert the moment object to a date, and compare the date objects:

```js
var newMin = scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) + minDelta);
options.time.min = moment(newMin).toDate() < moment(min_limit).toDate() ? min_limit : newMin;
```